doc-content/jbpm-docs/src/main/asciidoc/Kafka/message-send-event-proc.adoc

```diff
@@ -18,6 +18,13 @@ endif::JBPM,DROOLS,OP[]
 . Select the business process and open the business process designer.
 . Add an intermediate throw event or an end event of the type _message_ or _signal_.
 . Open the properties of the event.
-. In the *Message* or *Signal* field, select *New* and then enter the name of the message or signal. This name must be the same as the name of the topic to which the event is to send Kafka messages, or else must be defined in an `org.kie.server.jbpm-kafka.ext.topics.*` system property of the {KIE_SERVER}.
+. In the *Message* or *Signal* field, select *New* and then enter the name of the message or signal. This name must be the same as the name of the topic to which the event is to send Kafka messages, or else must be defined in an `org.kie.server.jbpm-kafka.ext.topics._broker-topic-name_` system property of the {KIE_SERVER}.
++
+For instructions about using `org.kie.server.jbpm-kafka.ext.topics.*` system properties to define topic names, see xref:kieserver-kafka-proc_{context}[].
++
 . Add an input data item. Select the data object that you created as its type.
-. Save the business process.
\ No newline at end of file
+. Save the business process.
+
+.Next steps
+
+To enable {KAFKA} integration when running the process, you must configure the {KIE_SERVER} according to instructions in xref:kieserver-kafka-proc_{context}[].
```

Comment:
Watch out for the `{KAFKA}` variable. It doesn't render anymore because you changed the variable to `{KAFKA_PRODUCT}` and for just plain "Kafka", you are just saying it explicitly. A quick search, assess, and replace would be good.